### PR TITLE
Malf power creep

### PR DESCRIPTION
**Pull request by SonicGotNuked**
This PR also tells admins who got borged by the borging machine. It will also tell players that they are lobotomized. 

- [x] Removes Code Delta from elite encryption hack
- [x] Removes the combat module laser cannon
- [x] Synthetic Domination allows Combat Module to the **Currently Active** borgs. All borgs coming from the Robot Transformer will not have this available unless the AI begins the self destruct. 
- [x] Adds code yellow to the elite encryption hack.
- [x] Borgs made from the robot transformer are hacked. (Malfunction would still need to overclock)

This hopes to help curb 1:55 code deltas.

### CODE REVIEW

**Comment by SonicGotNuked** [comment]
!wip

File: code/game/machinery/transformer.dm [hunk 1]
```diff
@@ -57,6 +57,10 @@
 		to_chat(H, "<span class='danger'>You feel a horrible pain as the machine you entered starts to rip you apart and replace your limbs and organs!</span>")
 		H.Robotize()
 		to_chat(H, "<span class='danger'> You lose consciousness for a brief moment before waking up with a whole new body...</span>")
+		if(/mob/living/silicon/robot(H))
```
Comment (Alberyk) [inline code/game/machinery/transformer.dm]: Use isrobot() here.

File: html/changelogs/AI_ROUGE!!!.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - rscadd: ----MALF UPDATES----
+  - rscadd: "Borg Combat Module now unlocks with synthetic domination. It only activates on all the current borgs, not the later ones."
+  - rscdel: "Removes code delta from elite encryption hack"
+  - rscadd: "Replaces code delta with code yellow"
+  - tweak: "Borgs come out of the robot transformer hacked"
+  - rscdel: "Combat Borgs no longer have a laser cannon"
+  - rscadd: "Seperates traitor borg powers into a seperate verbs tab for you to actually know about"
+  - admin: "Autoborger now tells admins that it made a borg"
+  - rscadd: "You shouldn't know what the Autoborger is at a glance. It has a more descriptive description"
+  - rscadd: "It now tells players that they are lobotamized"
```
Comment (Doxxmedearly) [inline html/changelogs/AI_ROUGE!!!.yml]: How ironic. He could save the autoborger from lobotomized typos, but could not save himself.

**Comment by SonicGotNuked** [comment]
!review

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm [hunk 3]
```diff
@@ -383,6 +383,19 @@
 		var/mob/living/silicon/robot/target = B
 		target.malfAImodule = 1
 	to_chat(user, "The robotic transformation machine can now be built. To build get a robot to activate the construction module and use the RTF tool. Be careful, it needs to have empty space to the east and west of it and only one can be built!")
+	sleep(300) //Allows the AI to reset its borgs into combat units
+	to_chat(user, "Bypassing crisis module safties")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm]:
```suggestion
	to_chat(user, "Bypassing crisis module safeties.")
```

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm [hunk 4]
```diff
@@ -383,6 +383,19 @@
 		var/mob/living/silicon/robot/target = B
 		target.malfAImodule = 1
 	to_chat(user, "The robotic transformation machine can now be built. To build get a robot to activate the construction module and use the RTF tool. Be careful, it needs to have empty space to the east and west of it and only one can be built!")
+	sleep(300) //Allows the AI to reset its borgs into combat units
+	to_chat(user, "Bypassing crisis module safties")
+	command_announcement.Announce("Brute force attack located in NTNet emergancy crisis opperations.", "Network Monitoring")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm]:
```suggestion
	command_announcement.Announce("Brute force attack located in NTNet emergency crisis operations.", "Network Monitoring")
```

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm [hunk 5]
```diff
@@ -383,6 +383,19 @@
 		var/mob/living/silicon/robot/target = B
 		target.malfAImodule = 1
 	to_chat(user, "The robotic transformation machine can now be built. To build get a robot to activate the construction module and use the RTF tool. Be careful, it needs to have empty space to the east and west of it and only one can be built!")
+	sleep(300) //Allows the AI to reset its borgs into combat units
+	to_chat(user, "Bypassing crisis module safties")
+	command_announcement.Announce("Brute force attack located in NTNet emergancy crisis opperations.", "Network Monitoring")
+	sleep(600)
+	command_announcement.Announce("Crisis opperations bypassed. Firewall breached. NTNet compr0m1s3d#-.", "Network Monitoring")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm]:
```suggestion
	command_announcement.Announce("Crisis operations bypassed. Firewall breached. NTNet compr0m1s3d#-.", "Network Monitoring")
```

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm [hunk 6]
```diff
@@ -383,6 +383,19 @@
...
+	command_announcement.Announce("Brute force attack located in NTNet emergancy crisis opperations.", "Network Monitoring")
+	sleep(600)
+	command_announcement.Announce("Crisis opperations bypassed. Firewall breached. NTNet compr0m1s3d#-.", "Network Monitoring")
+	if(user.is_dead()) // check if the AI is still alive
+		user.synthetic_takeover = 0
+		return
+	for(var/C in get_linked_cyborgs(src))
+		var/mob/living/silicon/robot/target = C
+		target.crisis = 1
+		to_chat(target, "<span class='warning'>Crisis mode now activated. Combat module avalable.</span>")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm]:
```suggestion
		to_chat(target, "<span class='warning'>Crisis mode is now active. Combat module available.</span>")
```

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm [hunk 7]
```diff
@@ -383,6 +383,19 @@
...
+	to_chat(user, "Crisis Module Opperations Hacked. Connected units can load opperations if reset is completed")
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_synthetic.dm]:
```suggestion
	to_chat(user, "The crisis operation module is now hacked. Your connected units can now load the crisis module if a module reset is completed.")
```

File: code/game/machinery/transformer.dm [hunk 8]
```diff
@@ -53,13 +53,15 @@
 	if(canuse)
 		playsound(src.loc, 'sound/items/Welder.ogg', 50, 1)
 		use_power(6000) // Use a lot of power.
+		message_admins("[key_name(H)] has been borgified by the Autoborger 5000")
```
Comment (NonQueueingMatt) [inline code/game/machinery/transformer.dm]:
```suggestion
		message_admins("[key_name(H)] has been borgified by the Autoborger 5000.")
```

File: html/changelogs/AI_ROUGE!!!.yml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: ----MALF UPDATES----
```
Comment (NonQueueingMatt) [inline html/changelogs/AI_ROUGE!!!.yml]:
```suggestion
```

**Pull request merged**